.github/workflows/image.yml

```diff
@@ -42,6 +43,7 @@ jobs:
             --build-arg BUILDKIT_INLINE_CACHE=1 \
             --build-arg SHOULD_BUILD_ADMIN_UI="$SHOULD_BUILD_ADMIN_UI" \
             --build-arg SHOULD_BUILD_RUST=false \
+            --build-arg ADMIN_SOURCEMAP_KEY="$ADMIN_SOURCEMAP_KEY" \
```

Comment:
as written this value will be in the image -- if it's actually secret this will leak the secret

Reply:
I didn't think of that. How would you recommend doing this?

Reply:
I think getsentry does this through the deploy pipeline -- I'm not entirely sure would have to check that

Reply:
I was thinking that too, but our images are built during CI, and I need this value during the build steps I think.

Reply:
is the value an actual secret? if not then we can just inline it

does it need to be in the source?

Reply:
I think it should be a secret? It's an internal integration API key for our Sentry projects. It would allow anyone with the key at minimum to upload files to our internal projects (that's what I plan to do with it).

Reply:
Can we use `docker build --secret id=mysecret,env=MYSECRET .`?

Reply:
I don't think that will help here -- the value is exposed in `ENV` and appears to get baked in by webpack

Reply:
Maybe I can have a separate step that builds webpack outside of the docker image just to get the sourcemaps uploaded. I don't need this value after the image is built.